incubator/hnc/pkg/controllers/object_controller_test.go

```diff
@@ -4,13 +4,12 @@ import (
 	"context"
 	"time"
 
+	tenancy "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
-
-	tenancy "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
```

Comment:
Can you restore this please? Usually we have (up to) three blocks of imports:

- Core libraries (eg `context`)
- Packages from other repos (eg `k8s.io/...`)
- Other packages from this repo (eg `v1alpha1`)